test/unit/pregnancy_registration.js

```diff
@@ -218,6 +220,52 @@ exports['valid adds lmp_date and patient_id'] = function(test) {
     });
 };
 
+exports['pregnancys on existing patients fail without valid patient id'] = function(test) {
```

Comment:
You haven't met Preg Nancy and her existing patients? Just wait until you meet Grand Preg Tarkin…


(I'll learn to spell somewhere in my 30s, I promise)